target_pos  : [4.0, -1.0, -2.0]
target_quat : [0,-0.7071068, 0, 0.7071068]
